src/cameras/cv_camera_manager.py

```diff
@@ -10,6 +10,7 @@
 from src.cameras.cam_singleton import get_or_create_cams
```

Comment:
The CV Camera Manager class allows upstream Developers to start a camera capture session, and returns a VideoWriter class (not to be confused with cv2.VideoWriter) that allows developers to write frame data without I/O penalties during processing.